### ISSUE TITLE
"Problem loading widget" on Android 10

### ISSUE THREAD

**Issue by mefjuu** [sAleksovski/react-native-android-widget#26]
First of all - big thanks for this useful library!

There is a problem on Android 10: after adding a widget, "WIDGET_UPDATE" action is fired. Sadly the widget on home screen shows "Problem loading widget" text though.
 
On Android 12 it's fine - the widget is shown properly.

Logcat says:

> ```2023-07-04 16:17:37.826  2901-2901  AppWidgetHostView       com.android.launcher3                W  Error inflating RemoteViews : android.widget.RemoteViews$ActionException: android.widget.RemoteViews$ActionException: android.widget.RemoteViews$ActionException: view: android.widget.FrameLayout can't use method with RemoteViews: setMinimumWidth(int)```

It seems to be a layout related problem. I was playing with elements removal and it appeared that the following code works:

```
   <FlexWidget
      style={{
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
      }}
    >
      <TextWidget style={{ fontSize: 12 }} text="Open App" />
    </FlexWidget>
```

But after adding another container the problem is back:

```
    <FlexWidget
      style={{
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
      }}
    >
      <FlexWidget
        style={{
          height: 'wrap_content',
          width: 'wrap_content',
          alignItems: 'center',
          justifyContent: 'center',
          padding: 16,
        }}
        clickAction="OPEN_APP"
      >
        <TextWidget style={{ fontSize: 12 }} text="Open App" />
      </FlexWidget>
    </FlexWidget>
```

**Comment by sAleksovski** [comment]
Hi,

Thanks for the detailed report and analysis. I'm glad people are finding it useful.

I think this was a regression introduced in `0.6.1`, and `0.6.0` should work.
The problem is not the second container, but the `clickAction` prop.

In `0.6.1`, adding `clickAction` will break the widgets on Android 10.

I'll try to fix this.

**Comment by sAleksovski** [comment]
I just released [`0.6.2`](https://github.com/sAleksovski/react-native-android-widget/releases/tag/v0.6.2) that should solve this.

Feel free to reopen the issue if you still have the error after upgrading.

**Comment by mefjuu** [comment]
Works like a charm. Thanks @sAleksovski !
